pkg/daemon/daemon.go

```diff
@@ -370,6 +396,43 @@ func (dn *Daemon) Close() {
 	dn.loginClient.Close()
 }
 
+// getMachineConfigFromFile parses a valid machine config file in yaml format and returns
+// a MachineConfig struct.
+func (dn *Daemon) getMachineConfigFromFile(filePath string) (*mcfgv1.MachineConfig, error) {
+	config := mcfgv1.MachineConfig{}
+	data, err := ioutil.ReadFile(filePath)
```

Comment:
Maybe we should add these ioutils to fsclient.go?:)

Reply:
Will do.

Reply:
https://github.com/openshift/machine-config-operator/pull/139